src/components/ha-sidebar-panel-config.ts

```diff
@@ -30,17 +31,19 @@ class HaSidebarPanelConfig extends LitElement {
 
   @state() private _issuesCount = 0;
 
-  static styles = [haStyleSidebarItem, styles];
-
   protected render() {
     const notices = this._updatesCount + this._issuesCount;
+    const keydown = createKeydown((e) =>
+      (e.currentTarget as HTMLElement).click()
+    );
+    const keyup = createKeyup((e) => (e.currentTarget as HTMLElement).click());
```

Comment:
This is still the same, as every time on render a new function is created and a new event listener is added. This should just be a class function.